bciers/apps/reporting/src/app/components/operations/personResponsible/PersonResponsible.tsx

```diff
@@ -44,10 +44,11 @@ const PersonResponsible = ({ version_id }: Props) => {
   const router = useRouter();
 
   const queryString = serializeSearchParams(useSearchParams());
-  const saveAndContinueUrl =
+  const continueUrl =
     operationType === "Linear Facility Operation"
       ? `/reports/${version_id}/facilities/lfo-facilities${queryString}`
       : `/reports/${version_id}/facilities/${facilityId}/review${queryString}&facilities_title=Facility`;
```

Comment:
There is an issue here with this path. When `queryString` is null, we get:
`.../review&facilities_title=Facility`, which doesn't parse properly because we don't have the `?` before the query params.

A potential solution (which would need to be investigated for side effects) is to change the [serializeSearchParams](https://github.com/bcgov/cas-registration/blob/develop/bciers/libs/utils/src/serializeSearchParams.ts#L11) utility function to return `?` when there are no params & append `&` to the end of the querystring when there are.

We'd have to look at where `serializeSearchParams` is used & make sure everything still works as expected where it's implemented

Reply:
To reproduce the bug, start a report & get to the Person Responsible page, once you enter data there & click Save&Continue, you'll get a 404 error because the params are missing the `?` to start them off